arviz/data/io_pyro.py

```diff
@@ -45,10 +66,18 @@ def __init__(
             self.nchains = self.ndraws = 0
```

Comment:
I have just seen the `if posterior is not None` and chain, draw definition is repeated below. This one can be removed